Lesson_5_Protractor/spec.js

```diff
@@ -0,0 +1,41 @@
+const { element, by, browser } = require('protractor');
+const { expect } = require('chai');
+
+describe('Protractor Demo App', function () {
+    const searchBar = element(by.xpath('//input[@type="search"]'));
+    const searchResults = element(by.css('.search-results'));
+    const ngDoc = element(by.xpath('//a[contains(@class,"search-result-item") and normalize-space()="ng doc"]'));
+    const ngDocHeader = element(by.css('header.cli-header'));
+    const ngDocArguments = element(by.xpath('//ul[@class="toc-list"]/li[normalize-space()="Arguments"]'));
+    const ngDocOptions = element(by.xpath('//ul[@class="toc-list"]/li[normalize-space()="Options"]'));
+    const introduction = element(by.css('aio-nav-menu.ng-tns-c18-1 > aio-nav-item:nth-child(1) > div > a'));
+    const angularBox = element(by.xpath('.//div[@class=\'card-container\']/a[@href=\'guide/what-is-angular\']'));
+    
+    browser.manage().window().setSize(1800, 600);
+  
+    beforeEach(function () {
+        browser.get('https://angular.io/');
+    });
+
+    browser.manage().timeouts().implicitlyWait(20000);
+
+    it('should have a result', () => {
+        searchBar.sendKeys('What is Angular');
+
+        expect(searchResults.isDisplayed()); 
```

Comment:
Ernest jan don't you need any kind of wait here?

Reply:
Laura jan when I type browser.manage().timeouts().implicitlyWait(20000);, it waits for all locators to appear.